javascript/config.yaml

```diff
@@ -35,7 +35,7 @@ language:
         - npm
       bootstrap:
         - npm --location=global install pm2
-      command: nohup pm2-runtime start "deno run --allow-net --allow-read=. app.ts" -i max
+      command: nohup pm2-runtime start "deno run --allow-env --allow-net --allow-read=. app.ts" -i max
```

Comment:
could you please explain this change ?

Reply:
I was getting an error, I guess deno 2 with npm:express requires `--allow-env` flag.

I would love to contribute on clustering deno btw